com.steamgriddb.steam-rom-manager.json

```diff
@@ -0,0 +1,85 @@
+{
+	"app-id": "com.steamgriddb.steam-rom-manager",
+	"default-branch": "stable",
+	"runtime": "org.freedesktop.Sdk",
+	"runtime-version": "21.08",
+	"sdk": "org.freedesktop.Sdk",
+	"base": "org.electronjs.Electron2.BaseApp",
+	"base-version": "21.08",
+	"command": "steam-rom-manager",
+	"separate-locales": false,
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--device=all",
+		"--filesystem=host",
+		"--env=IN_FLATPAK=1"
+	],
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/share/gtk-doc",
+		"*.la"
+	],
+	"modules": [
+		{
+			"name": "zypak",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/refi64/zypak",
+					"tag": "v2022.03",
+					"commit": "8424c6bcf909ca2031ebc6537d324bb64772266e"
+				}
+			]
+		},
+		{
+			"name": "steam-rom-manager",
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -D run.sh /app/bin/steam-rom-manager",
+				"cp /usr/bin/ar /app/bin",
+				"ar x srm.deb",
+				"tar xf data.tar.xz",
+				"mv \"opt/Steam ROM Manager\" \"/app/srm\"",
+				"rm -rf \"usr/share/icons/hicolor/1024x1024\"",
+				"cd usr; find share/icons/hicolor -type f -exec install -Dm644 \"{}\" \"/app/{}\" \\;",
+				"find /app/share/icons/hicolor -name \'*.png\' -type f -exec bash -c \'mv \"$1\" \"${1/\\/steam-rom-manager/\\/com.steamgriddb.steam-rom-manager}\"\' -- {} \\;",
```

Comment:
you can just use `rename-icon: steam-rom-manager` (top-level of the manifest)

Reply:
🤦‍♂️

Reply:
I spent like 2 hours trying to figure out how to do that lol